src/Token/_RemoveTokenButton.tsx

```diff
@@ -0,0 +1,28 @@
+import {XIcon} from '@primer/octicons-react'
+import styled from 'styled-components'
+import {defaultTokenSize} from './TokenBase'
+import {TokenButtonProps, tokenButtonStyles, variants, getTokenButtonIconSize} from './_tokenButtonUtils'
+
+const RemoveTokenButton = styled.span.attrs<TokenButtonProps>(
+  ({borderOffset, parentTokenTag, variant, parentTokenIsInteractive, ...rest}) => {
+    delete rest.children
+
+    return {
+      borderOffset,
+      as: parentTokenIsInteractive ? 'span' : 'button',
```

Comment:
Curious when we would want this button to be rendered as a span?

Reply:
Ah, this is to avoid rendering a button inside an interactive element.

Reply:
In the storybook, clicking the remove icon on the link token takes me to the link address. Is that what we want?
![CleanShot 2021-10-04 at 12 00 03](https://user-images.githubusercontent.com/4608155/135908825-3b2b978d-da7f-4e78-ac3d-5af81fac9561.gif)

Reply:
> In the storybook, clicking the remove icon on the link token takes me to the link address. Is that what we want?

No, this is a bug. Will fix